scripts/archive-status

```diff
@@ -12,13 +12,13 @@ if [[ -z $1 ]]; then
     usage
     exit 1
 fi
-    
+
 if [[ ($1 == "--help") || ($1 == "-h") ]]; then
     usage
     exit 0
 fi
 
 result=$(curl -s "http://pscaa01.slac.stanford.edu:17665/mgmt/bpl/getPVStatus?pv=$1")
```

Comment:
For followup: these urls will be deprecated soon.  We should be using `https://pswww.slac.stanford.edu/archiveviewer/retrieval/`